api-reference/10 UI Widgets/dxGantt/1 Configuration/validation/autoUpdateParentTasks.md

```diff
@@ -5,7 +5,15 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Enables parent-child task relationships validation.
```

Comment:
It seems that the option's name doesn't fit the description. Looking at the name `autoUpdateParentTasks`, I assume that whenever a user edits a child task (changes its duration, start or end time, etc.), its parent task automatically adjusts to the changes. However, the description talks about validation. Validation means that something is simply checked for validity, and no futher action is done. So, how does this option work in reality?

Reply:
Updated

Reply:
It works like you've described. Automatically updates.